src/api/CortexClient.ts

```diff
@@ -401,7 +401,7 @@ export class CortexClient implements CortexApi {
 
     const headers = {
       ...init?.headers,
-      Authorization: `Bearer ${token}`,
+      Authorization: `Bearer ${token.replace(/[Bb]earer\s+/, '').trim()}`,
```

Comment:
How about
```suggestion
      Authorization: `Bearer ${token.replace(/^[Bb]earer\s+/, '').trim()}`,
```
just to be paranoid?